packages/miniapp-registry/src/index.ts

```diff
@@ -9,12 +9,14 @@ import NFTMinter from "@miniapps/nft-minter";
 import UrlRender from "@miniapps/url-render";
 import ImageRender from "@miniapps/image-render";
 import ChatGPTShorten from "@miniapps/chatgpt-shorten";
+import ZoraNftMinter from "@miniapps/zora-nft-minter";
 
 export const allMiniApps = [
   InfuraIPFSUpload,
   LivepeerVideo,
   GiphyPicker,
   VideoRender,
+  ZoraNftMinter,
```

Comment:
think we should hold off of this for now; or not publish a new version

Reply:
excluded from registry for now